### ISSUE TITLE
Show series score in GameDay CV during elims

### ISSUE THREAD

**Issue by andrewda** [the-blue-alliance/the-blue-alliance#2129]
Showing alliance number would probably go hand-in-hand with this.

**Comment by fangeugene** [comment]
Good idea. Calculating series score client-side goes against our whole "thin clients" idea, but we don't have any API endpoints that return alliance status (except for the Team@Event ones).
